witchcraft/server_metrics.go

```diff
@@ -130,14 +127,33 @@ func (s *Server) initMetrics(ctx context.Context, installCfg config.Install) (rR
 		localMetricLogger.Metric(metricID, metricType, metric1log.Values(valuesToUse), metricTagsParam)
 	}
 
+	emitAllFn := func() {
+		// emit all metrics a final time on termination
```

Comment:
I don't think this comment makes sense -- doesn't `emitAllFn` get called both periodically while alive *and* on termination?

Reply:
Yes, apologies, I incorrectly copied this from the function literal below. Removed